start_esp/start_esp.py

```diff
@@ -460,6 +468,10 @@ def make_argparser():
         header, Default value: {xff_trusted_proxy_list}'''.
         format(xff_trusted_proxy_list=DEFAULT_XFF_TRUSTED_PROXY_LIST))
 
+    parser.add_argument('--check_metadata', action='store_true',
```

Comment:
what is the default value?

Reply:
The default value is False. store_true action is handling it.